packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -217,8 +217,14 @@ export type SubmitSummaryResult =
 /** The stages of Summarize, used to describe how far progress succeeded in case of a failure at a later stage. */
 export type SummaryStage = SubmitSummaryResult["stage"] | "unknown";
 /** The data in summarizer result when submit summary stage fails. */
```

Comment:
Put this back with the right type

Reply:
I reverted most of the changes here. Please take a look.